src/collective/collectionfilter/vocabularies.py

```diff
@@ -140,6 +146,16 @@ def groupby(self):
             return self._groupby
         self._groupby = {}
 
+        # Use 'try' since it broke before update
+        # Maybe move this code to GroupByCriteria
+        try:
+            INTEGER_IDXS = plone.api.portal.get_registry_record(
+                "collective.collectionfilter.int_types", default=[]
+            )
+        except ComponentLookupError:
```

Comment:
I thought registry lookup errors are handled in `plone.api` ... and please follow the suggestion of the docs:

> Always import the top-level package (from plone import api) and then use the module namespace to access the method you want (portal = api.portal.get()).


See https://6.docs.plone.org/plone.api/about.html#design-decisions